### PR TITLE
Converting the wiki to documentation

### PR DESCRIPTION
**Pull request by zimbatm**
See https://github.com/NixOS/nixpkgs/issues/13180

I'm importing the examples in the modules doc. It doesn't do justice to a proper guide though.

### CODE REVIEW

**Comment by garbas** [comment]
@zimbatm: does this also closes  #13281 (wiki: Samba)?

and please don't despair, once the content is moved into docbook .. we'll focus on the structure and better discoverability, i promise!! :)

**Comment by zimbatm** [comment]
hehe yes it does. Updates the commit to point over there.

File: nixos/modules/services/hardware/sane.nix [hunk 1]
```diff
@@ -21,7 +21,11 @@ in
     hardware.sane.enable = mkOption {
       type = types.bool;
       default = false;
-      description = "Enable support for SANE scanners.";
+      description = ''
+        Enable support for SANE scanners.
+
+        Users in the "scanner" group will gain access to the scanner.
```
Comment (edolstra) [inline nixos/modules/services/hardware/sane.nix]: Note that paragraph breaks like this won't show up in the generated docs. I think adding `<para>` does work.

Comment (zimbatm) [inline nixos/modules/services/hardware/sane.nix]: Thanks, fixed

**Pull request merged**